content/en/agent/guide/network.md

```diff
@@ -56,7 +58,8 @@ All of these domains are **CNAME** records pointing to a set of static IP addres
 
   - [APM][1] data is `trace.agent.datadoghq.eu`
   - [Live Containers][2] data is `process.datadoghq.eu`
-  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][5] for more information.
+  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][6] for more information.
+  - [Orchestrator Resources][4] data is `orchestrator.datadoghq.com`
```

Comment:
@hkaj Wouldn't this be .eu? Want to verify before committing a change.

Also just a heads up, there was a weird merge commit error with master as we just pushed changes this afternoon to this same doc. Added what was in that PR to this PR so we don't overwrite those changes.